app/uk/gov/hmrc/gform/sharedmodel/TaxPeriods.scala

```diff
@@ -70,9 +74,32 @@ object TaxPeriodInformation {
 
 sealed trait Obligations
 final case object NotChecked extends Obligations
-final case class RetrievedObligations(listOfObligations: NonEmptyList[TaxPeriodInformation]) extends Obligations
+final case class RetrievedObligations(listOfObligations: List[TaxPeriodInformation]) extends Obligations
 
 object Obligations {
-  import JsonUtils._
   implicit val format: OFormat[Obligations] = derived.oformat
 }
+
+case class IdNumberValue(value: String) extends AnyVal
```

Comment:
do we need "extends AnyVal" maybe we can avoid it.

Reply:
As far as I know when we have a case class that takes in only a single basic parameter we are meant to extend AnyVal. I'm not too sure why so if you have a reason to remove it we can